docs/DeveloperGuide.adoc

```diff
@@ -800,12 +801,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
+|`* * *` |user |schedule meetings with my clients |remember when I am supposed to meet them
```

Comment:
You may want to be more specific than just `user`. 
Think what _kind of user_ would benefit from the feature.